composer.json

```
@@ -9,20 +9,20 @@
     "docs": "https://googleapis.github.io/google-auth-library-php/main/"
   },
   "require": {
-    "php": "^7.1||^8.0",
-    "firebase/php-jwt": "^5.5||^6.0",
+    "php": "^7.4||^8.0",
+    "firebase/php-jwt": "^6.0",
     "guzzlehttp/guzzle": "^6.2.1|^7.0",
-    "guzzlehttp/psr7": "^1.7|^2.0",
+    "guzzlehttp/psr7": "^2.0",
     "psr/http-message": "^1.0||^2.0",
```

[Optional] Seems [v1.0](https://packagist.org/packages/psr/http-message#1.0) supports php<7.4, so maybe we are better with 1.1
```suggestion
    "psr/http-message": "^1.1||^2.0",
```

---

SGTM